shared/src/test/diff/codegen/ConstructorStmt.mls

```diff
@@ -185,5 +184,58 @@ class Baz {
 //│   let x: 123
 //│   let y: int
 //│ }
-//│ Code generation encountered an error:
-//│   unresolved symbol x
+//│ // Prelude
+//│ let typing_unit9 = {
+//│   cache: {},
+//│   get Baz() {
+//│     const cache = this.cache;
+//│     if (this.cache.Baz === undefined) {
+//│       class Baz {
+//│         #x;
+//│         #y;
+//│         get x() { return this.#x; }
+//│         get y() { return this.#y; }
+//│         constructor() {
+//│           this.#x = 123;
+//│           const x = this.#x;
+//│           log([
+//│   1,
```

Comment:
Do you know why the indentation is bad? Can we easily fix it?

Reply:
I'm trying to figure it out